src/main/scala/org/allenai/pipeline/Pipeline.scala

```diff
@@ -76,17 +105,20 @@ trait Pipeline extends Logging {
       ).signature
       s"${signature.name}.${signature.id}$suffix"
     }
-    implicitly[ClassTag[A]].runtimeClass match {
-      case c if c == classOf[FlatArtifact] => original.persisted(
-        io,
-        artifactFactory.flatArtifact(s"data/$path").asInstanceOf[A]
-      )
-      case c if c == classOf[StructuredArtifact] => original.persisted(
-        io,
-        artifactFactory.structuredArtifact(s"data/$path").asInstanceOf[A]
-      )
-      case _ => sys.error(s"Cannot persist using io class of unknown type $io")
-    }
+    val persisted =
```

Comment:
Check to see if output path is absolute URL before falling back to a relative path.